examples/advanced_operations/add_sitelink.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Demonstrates how to create a sitelink extension."""
+
+from __future__ import absolute_import
+
+import argparse
+import six
+import sys
+import uuid
+
+import google.ads.google_ads.client
+
+
+def main(client, customer_id):
+    # Create an extension setting.
+    feed_service = client.get_service('ExtensionFeedItemService', version='v1')
+
+    extension_feed_item_operation = client.get_type('ExtensionFeedItemOperation', version='v1')
+    extension_feed_item = extension_feed_item_operation.create
+    extension_feed_item.sitelink_feed_item.link_text.value = 'Text'
+    extension_feed_item.sitelink_feed_item.line1.value = 'Line 1 Value'
+    extension_feed_item.sitelink_feed_item.line2.value = 'Line 2 Value'
+    final_url = extension_feed_item.sitelink_feed_item.final_urls.add()
+    final_url.value = 'www.example.com'
+
+    # Add extension
+    try:
+        feed_response = (
+            feed_service.mutate_extension_feed_items(customer_id, [extension_feed_item_operation])
+        )
+    except google.ads.google_ads.errors.GoogleAdsException as ex:
```

Comment:
Similar to my comment about `GoogleAdsClient` can you add this to the top of the file:

```
from google.ads.google_ads.errors import GoogleAdsException
```
Then this line can just be:

```
except GoogleAdsException as ex:
```

Reply:
also much better, we should adjust in the other examples as well. I can help, if you'd like me to.